### ISSUE TITLE
Command titles in the keybinding contexts

### ISSUE THREAD

**Issue by Chinggis6** [The-Compiler/qutebrowser#2270]
It is a feature request.

I have a lot of my own key bindings and I sort them under contexts.

For example I have a separate context for **W**ikipedia and **V**ideo (`mpv` player), as seen below.

![Wikipedia context](http://i.imgur.com/ftpmD14.png)

![Video context](http://i.imgur.com/tWvpYBe.png)

I'd like, when binding, to additionaly (perhaps with `bind --title ""`, as an optional switch) put my titles (or tooltips) for the commands binded, so it's more user-friendly and easier to read, understand the function of, and remember.

It's especially useful when binding verbose commands into a key, or when userscripts are involved.

I think, it should be an optional feature for the context, and might be implemented as the middle column of the tabular context which can be empty.

**Comment by The-Compiler** [comment]
cc @rcorre - probably one for after the config revolution though.

**Comment by rcorre** [comment]
yep, probably best for after #499. Which I actually want to look at after #74. Which is coming soon! For real this time!

@Chinggis6, In the UI, would you expect the title to come before/after the action, or to replace the action text entirely? In other words, would you expect `Vv spawn mpv {url} Open in mpv` or `Vv Open in MPV`? I'm thinking the latter because it is easier to read at a glance.

**Comment by The-Compiler** [comment]
I also think the text alone would suffice - and we could also provide a neat default text for the default bindings.

I'd sure appreciate a bit of help with #499, but we need to carefully coordinate that - please join the IRC channel first and let's discuss before you start any work on that :wink:

**Comment by pkill-nine** [comment]
just a note - it would be preferable for the title not to be mandatory :)
